tests/sflow_test.go

```diff
@@ -105,7 +105,19 @@ func (s *TestStorage) SearchFlows(filters storage.Filters) ([]*flow.Flow, error)
 	return nil, nil
 }
 
-func (s *TestStorage) CheckFlow(t *testing.T, f *flow.Flow, trace *flowsTraceInfo) bool {
+func (s *TestStorage) GetFlows() []*flow.Flow {
+	s.lock.Lock()
+	defer s.lock.Unlock()
+
+	flows := make([]*flow.Flow, 0, len(s.flows))
```

Comment:
As we known the size of the final list, why not initialized to the right value instead of 0